Tests/FeaturevisorSDKTests/InstanceTests.swift

```diff
@@ -590,5 +590,84 @@ class FeaturevisorInstanceTests: XCTestCase {
         XCTAssertTrue(refreshed)
         XCTAssertTrue(updatedViaOption)
     }
+    
+    func testShouldStartRefreshing() {
+
+        // GIVEN
+        var revision = 1
+        var refreshedCount = 0
+        let refreshInterval = 1
+        let expectedRefreshCount = 3
+        let maxWaitTime = TimeInterval(expectedRefreshCount) * TimeInterval(refreshInterval) + 1
+
+        let expectation: XCTestExpectation = expectation(description: "Expectation")
+        var expectationFulfilled = false
```

Comment:
Do we really need it? Can we just relay on expectedRefreshCount?

Reply:
I mean otherwise the code finishes too fast, I can change it to be dependant on some while loop instead.